parts/kubernetesagentcustomdata.yml

```diff
@@ -135,7 +135,8 @@ write_files:
             --allow-privileged=true \
             --enable-server \
             --enable-debugging-handlers \
-            --config=/etc/kubernetes/manifests \
+            --feature-gates=Accelerators=true \
```

Comment:
This flag is only used for GPU discovery currently (might be used for other type of hardware discovery in the future).
If a VM has this flag enabled, but no GPU is present, nothing really happens. The list of available GPU will be empty and no GPU workloads will be scheduled on it as you see [here](https://github.com/kubernetes/kubernetes/blob/ed479163fa3772895be6a837d0b645ec62bfd383/pkg/kubelet/gpu/nvidia/nvidia_gpu_manager.go#L212).
No error will be logged either as far as I can tell, so that should be transparent for non-GPU customers.  
  
The only thing is that if the container engine used is not docker, then an error message will get displayed and the accelerator flag will be disabled, but anyway I don't think we support anything else than docker? (see [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L804))